projects/fsastorefrontlib/src/cms-components/b2b/units.config.ts

```diff
@@ -0,0 +1,211 @@
+};
+
+export const unitsRoutingConfig: RoutingConfig = {
+  routing: {
+    routes: {
+      orgUnitProductAssignment: {
+        paths: [`${listPath}/unitProductAssignments`],
+        paramsMapping,
+      },
+      orgUnitProductAssign: {
+        paths: [`${listPath}/unitProductAssignments/assign`],
+        paramsMapping,
+      },
+    },
+  },
+};
+
+export const unitsCmsConfig: CmsConfig = {
+  cmsComponents: {
+    ManageUnitsListComponent: {
+      component: UnitListComponent,
+      providers: [
+        {
+          provide: ListService,
+          useExisting: UnitListService,
+        },
+        {
+          provide: ItemService,
+          useExisting: UnitItemService,
+        },
+      ],
+      childRoutes: {
+        parent: {
+          data: {
+            cxPageMeta: {
+              breadcrumb: 'orgUnit.breadcrumbs.list',
+            },
+          },
+        },
+        children: [
+          {
+            path: 'create',
+            component: UnitFormComponent,
+          },
+          {
+            path: `:${ROUTE_PARAMS.unitCode}`,
+            component: FSUnitDetailsComponent,
+            data: {
+              cxPageMeta: { breadcrumb: 'orgUnit.breadcrumbs.details' },
+            },
+            children: [
+              {
+                path: 'edit',
+                component: UnitFormComponent,
+              },
+              {
+                path: 'children',
+                component: UnitChildrenComponent,
+                data: {
+                  cxPageMeta: { breadcrumb: 'orgUnit.breadcrumbs.children' },
+                },
+                children: [
+                  {
+                    path: 'create',
+                    component: UnitChildCreateComponent,
+                  },
+                ],
+              },
+              {
+                path: 'approvers',
+                data: {
+                  cxPageMeta: { breadcrumb: 'orgUnit.breadcrumbs.approvers' },
+                },
+                children: [
+                  {
+                    path: '',
+                    component: UnitAssignedApproverListComponent,
+                  },
+                  {
+                    path: 'assign',
+                    component: UnitApproverListComponent,
+                  },
+                ],
+              },
+              {
+                path: 'users',
+                component: UnitUserListComponent,
+                data: {
+                  cxPageMeta: { breadcrumb: 'orgUnit.breadcrumbs.users' },
+                },
+                children: [
+                  {
+                    path: 'create',
+                    component: UnitUserCreateComponent,
+                  },
+                  {
+                    path: `:${ROUTE_PARAMS.userCode}/roles`,
+                    component: UnitUserRolesFormComponent,
+                  },
+                ],
+              },
+              {
+                path: 'unitProductAssignments',
```

Comment:
Is it possible to just add this to configuration, without copying everything above ? Since it is the same as in original "unitsCmsConfig" ?

Reply:
It's not, and it's not the same since we are using our `FSUnitDetailsComponent` instead of Sparatacus' `UnitDetailsComponent`. Unfortunately, even if that were not the case, because we are overriding the entire Spartacus units.config with this one, we need to specify which component is tied to which route.